Modules/timemodule.c

```diff
@@ -586,8 +586,11 @@ time_strftime(PyObject *self, PyObject *args)
         if (_PyTime_localtime(tt, &buf) != 0)
             return NULL;
     }
-    else if (!gettmarg(tup, &buf) || !checktm(&buf))
+    else if (!gettmarg(tup, &buf, "iiiiiiiii;strftime(): illegal time tuple "
```

Comment:
Wouldn't it look better if break a line before the format argument and before `!checktm(&buf)`?